### PR TITLE
Use PIL and cv2 for rotate

### PR DESCRIPTION
**Pull request by yuyu2172**
Merge after #739

### CODE REVIEW

File: chainercv/transforms/image/rotate.py [hunk 1]
```diff
@@ -1,26 +1,72 @@
...
+        xx = []
+        for y, x in ((0, 0), (H, 0), (H, W), (0, W)):
+            yy.append(
+                affine_mat[1, 0] * x + affine_mat[1, 1] * y + affine_mat[1, 2])
+            xx.append(
+                affine_mat[0, 0] * x + affine_mat[0, 1] * y + affine_mat[0, 2])
+        out_H = int(np.ceil(max(yy)) - np.floor(min(yy)))
+        out_W = int(np.ceil(max(xx)) - np.floor(min(xx)))
+
+        affine_mat[1][2] += out_H / 2 - H / 2
```
Comment (knorth55) [inline chainercv/transforms/image/rotate.py]: `affine_mat[1, 2] = out_H / 2`?

Comment (knorth55) [reply chainercv/transforms/image/rotate.py]: sorry, I misunderstood.

File: chainercv/transforms/image/rotate.py [hunk 2]
```diff
@@ -1,26 +1,72 @@
...
+
+def _rotate_pil(img, angle, expand, fill, interpolation):
+    out = []
+    for ch in img:
+        ch = PIL.Image.fromarray(ch, mode='F')
+        out.append(np.array(
+            ch.rotate(
+                angle, expand=expand,
+                fillcolor=fill, resample=interpolation)))
+    return np.stack(out).astype(img.dtype)
```
[the remark this answers is not in the document]
Comment (yuyu2172) [reply chainercv/transforms/image/rotate.py]: Are you saying it should be `np.rounded` in the case when `dtype` is `int`?
Or, are you saying, it should be rounded all the time.

[the remark this answers is not in the document]
Comment (yuyu2172) [reply chainercv/transforms/image/rotate.py]: I see. That makes sense.

[the remark this answers is not in the document]
Comment (yuyu2172) [reply chainercv/transforms/image/rotate.py]: fixed

**Comment by yuyu2172** [comment]
@knorth55

**Comment by knorth55** [comment]
LGTM

**Pull request merged**